pkg/containerd/cmd/service/system_init.go

```diff
@@ -71,6 +76,32 @@ func systemInit(args []string) {
 		os.Exit(1)
 	}
 
+	// remove (unlikely) old containerd socket
+	_ = os.Remove(*sock)
+
+	// start up containerd
+	cmd := exec.Command(*binary)
+	cmd.Stderr = os.Stderr
```

Comment:
It would be useful to send stdout somewhere, perhaps to `/var/log/system-containerd.log`? Previously all stdio went to the console so doing `cmd.Stdout = os.Stdout` too would retain the status quo.